### ISSUE TITLE
Fix waitFor correctly

### ISSUE THREAD

**Issue by Katsute** [Ktt-Development/rexedia#5]
Fix thread hold issue on `#waitFor` by checking if last line was a terminate batch requests (y/n) instead of using a future

**Comment by Katsute** [comment]
This issue only occurs on the CI for some reason

**Comment by Katsute** [comment]
https://github.com/Ktt-Development/rexedia/blob/361b6446b80fd88641bf6adbf3bfc9cbcb88991e/src/main/java/com/kttdevelopment/rexedia/format/CommandExecutor.java#L50-L71

CI freezes after line 59 despite there being no possible way this method can freeze at that point.

**Comment by Katsute** [comment]
CI Logs for reference
```
Sun, 04 Oct 2020 18:01:43 GMT
[FINEST] [26@com.kttdevelopment.rexedia.format.CommandExecutor#lambda$execute$0] b4 future
Sun, 04 Oct 2020 18:01:43 GMT
[FINER] [26@com.kttdevelopment.rexedia.format.CommandExecutor#lambda$execute$0]   configuration: ...
Sun, 04 Oct 2020 18:01:43 GMT
[FINEST] [26@com.kttdevelopment.rexedia.format.CommandExecutor#lambda$execute$0] ar future
Sun, 04 Oct 2020 18:07:17 GMT
Terminate batch job (Y/N)? 
Sun, 04 Oct 2020 18:07:18 GMT
Error: The operation was canceled.
```

**Comment by Katsute** [comment]
🎉 Fixed in 74a0556
